velox/connectors/hive/FileHandle.h

```diff
@@ -29,11 +29,11 @@
 #include <memory>
 #include <string>
 
+#include "dwio/common/InputStream.h"
```

Comment:
Let's keep the full path here. Otherwise it breaks the build system on Meta side

Reply:
@zzhao0 Thanks for reviewing the PR. I actually had this fixed before but the change was lost. Now the new commit contains the fix.